src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -24,6 +25,17 @@ public class VerifyCommandRunner : IVerifyCommandRunner
         private const int SuccessCode = 0;
         private const int FailureCode = 1;
         private const HashAlgorithmName _defaultFingerprintAlgorithm = HashAlgorithmName.SHA256;
+        private const string TrustedSignersSectionName = "trustedSigners";
+        private ISettings _settings;
+
+        [Obsolete("This API does ignores NuGet.Config file or trusted-signers list")]
+        public VerifyCommandRunner()
+        { }
+
+        public VerifyCommandRunner(ISettings settings)
```

Comment:
If you look at `RestoreCommandRunner` as a point of comparison, it does not take an `ISettings` as a constructor parameter, instead its `ExecuteCommandAsync` takes a `RestoreArgs`, which contains a `ConfigFile` property. Therefore, I believe that the settings should be moved to `VerifyArgs`

Reply:
Sure, moved to `VerifyArgs`.